fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LuceneIndexExpressions.java

```diff
@@ -158,14 +159,14 @@ public boolean isSorted() {
         public PointsConfig getPointsConfig() {
             switch (type) {
                 case INT:
-                    return new PointsConfig(NumberFormat.getInstance(), Integer.class);
+                    return new PointsConfig(NumberFormat.getInstance(Locale.ROOT), Integer.class);
```

Comment:
Question: Does that have any implication on the way numbers are formatted and parsed from a user perspective? What does ROOT locale do that is different than Locale.ENGLISH or Locale.US? 
Lucene support in general is restricted to English (see stop words for example). Does ROOT match that definition?